README.md

```diff
@@ -27,7 +27,17 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
 
 ## Installing the plugin into your IDE
 
-(coming soon)
+If you'd like to try out the plugin pre-release, you can build it from source and install it into your IDE:
+
+1) Clone this repository to your machine.
```

Comment:
Insert `git clone ...` string here just for easy copy-paste step-by-step instructions?

Reply:
added